scripts/test262-runner.js

```diff
@@ -1098,7 +1101,9 @@ function testFilterByMetadata(
   if (test.location.includes("Simd")) return false;
 
   // temporarily disable intl402 tests (ES5)
-  if (test.location.includes("intl402")) return false;
+  if (test.location.includes("intl402") && !test.location.includes('/Date/')) {
```

Comment:
This brings in some of the intl402 date tests but the tests don't appear to be related to `Date.parse` as far as I can tell. I noticed some of the additional tests pass and some fail (all failures appear to be related to `Date.Date.prototype.toLocaleString()` not being  implemented).